src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -112,6 +115,40 @@ private CallbackValidations() {
             : invalid("Exception case has no Id");
     }
 
+    @Nonnull
+    public static Validation<Seq<String>, CaseDetails> hasValidDetailsForAttachingToCase(
```

Comment:
is this used for validating prerequisites for updating case with OCR + Scanned docs ?
can we name it accordingly `hasValidDetailsForAttachingOCRWithSuppEvidence` ?

Reply:
The whole point of this PR is to introduce this method which combines all checks called from AttachCaseCallbackService::getValidation method. In the existing code all this check can be invoked separately because to just attach documents we need these documents and case ID. But for this user story (update case) we need the whole CaseDetails object (i.e. exception record) to send it to the service Update endpoint. None of existing validation methods returns the whole CaseDetails object. This is why this new method has been introduced - it combines all validations and return the whole CaseDetails object.

After this PR is merged the new validation method will be called from AttachCaseCallbackService::getValidation. And AttachCaseCallbackService::AttachToCaseEventData will contain the whole CaseDetails object (returned by the new validation method) instead of just exceptionRecordId.

Reply:
No, it is not specific to OCR, it is for any supplementary Evidence